sbom/formatted_reader.go

```diff
@@ -53,6 +54,30 @@ func (f *FormattedReader) Read(b []byte) (int, error) {
 			return 0, fmt.Errorf("failed to format sbom: %w", err)
 		}
 
+		// Makes CycloneDX SBOM more reproducible, see
+		// https://github.com/paketo-buildpacks/packit/issues/367 for more details.
+		if f.format.ID() == "cyclonedx-1.3-json" || f.format.ID() == "cyclonedx-1-json" {
```

Comment:
Why `cyclonedx-1-json`?

Reply:
That is the generic ID term for cycloneDX 1.4